.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: node_js
 node_js:
+  - "14"
   - "12"
-  - "10"
 addons:
   code_climate:
     repo_token: 2ae0377f20bf6db6d94dc02470f60a0dae8cf2862e2fe7fe092472dd4e2978cf
 after_script:
-  - npm run lcov
```

Comment:
I'm presuming that this is no longer required because you've switched to jest? I'm unsure what we think of jest vs mocha.